package.json

```diff
@@ -24,14 +24,15 @@
   "dependencies": {
     "ember-apollo-client": "2.0.0-beta.1",
     "ember-auto-import": "1.2.13",
-    "ember-caluma-utils": "anehx/ember-caluma-utils#save-answer",
+    "ember-caluma-utils": "anehx/ember-caluma-utils#default-options",
```

Comment:
Shall we leave this as is or can you change to `#master` already?

Reply:
I'll change this to master